metadata.txt

```diff
@@ -3,17 +3,18 @@
 # This file should be included when you package your plugin.# Mandatory items:
 
 [general]
-name=Mapping Ecosystem Services
-qgisMinimumVersion=3.0
-description=This plugin maps ecosystems
+name=Mapping Biocontrol Ecosystem Services
```

Comment:
@lcalisto I guess this makes necessary change the name in a few places here https://github.com/NaturalGIS/mapping_ecosystem_services/blob/master/mapping_ecosystem_services.py ?